src/main/java/frc/robot/subsystems/turretSubsystem/TurretSubsystem.java

```diff
@@ -441,6 +441,14 @@ public void aimTurret() {
           tagDistanceEntry.log(distanceToTag);
         }
 
+        if (visionSubsystem.getTurretPitch(tagID) == 0) {
```

Comment:
Let's not call this twice, but instead call it once, and save the value into a local variable

Reply:
Can we add a comment on what does it mean to have getTurretPitch == 0?

It will help when we read this code later and try to understand what is happening.